x-pack/test/security_solution_cypress/package.json

```diff
@@ -28,6 +28,7 @@
     "cypress:changed-specs-only:serverless": "yarn cypress:serverless --changed-specs-only --env burn=5",
     "cypress:burn:serverless": "yarn cypress:serverless --env burn=2",
     "cypress:qa:serverless": "TZ=UTC NODE_OPTIONS=--openssl-legacy-provider node ../../plugins/security_solution/scripts/start_cypress_parallel_serverless --config-file ../../test/security_solution_cypress/cypress/cypress_ci_serverless_qa.config.ts",
+    "cypress:open:qa:serverless": "yarn cypress:qa:serverless open",
```

Comment:
I'm curious about command naming. We have only one separate QA environment we run tests against. It turns out `qa` is a subset of `serverless` in some sense. I can imagine we have ESS QA environment in the future and this naming will have more sense.

Have you considered the other options like `cypress:open:serverless-qa`?

Naming in this file isn't consistent but we can improve it by sticking to one format something like `cypress:mode:env[:suites name]`.